js/main.js

```diff
@@ -0,0 +1,22 @@
+const submit = document.querySelector('#submit');
+const form = document.querySelector('form');
+const navIcon = document.querySelector('#nav-icon');
+const nav = document.querySelector('nav');
+
+form.addEventListener('submit', (e)=> {
```

Comment:
`navIcon` doesn't seem to exist in any of your pages - did you forget to add it in?